tests/SpecTests/RetryableWritesSpecTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace MongoDB\Tests\SpecTests;
+
+/**
+ * Retryable writes spec tests.
+ *
+ * @see https://github.com/mongodb/specifications/tree/master/source/retryable-writes
+ */
+class RetryableWritesSpecTest extends FunctionalTestCase
+{
+    /**
+     * Execute an individual test case from the specification.
+     *
+     * @dataProvider provideTests
+     * @param string $name  Test name
+     * @param array  $test  Individual "tests[]" document
+     * @param array  $runOn Top-level "runOn" document
+     * @param array  $data  Top-level "data" array to initialize collection
+     */
+    public function testRetryableWrites($name, array $test, array $runOn = null, array $data)
+    {
+        $this->setName($name);
+
+        if (isset($runOn)) {
+            $this->checkServerRequirements(
```

Comment:
This is incorrectly implemented, since `runOn` is an array of requirements and we should only skip if none are satisfied. Will revise in a subsequent commit.